tests/phpunit/tests/formatting/maybeHashHexColor.php

```diff
@@ -7,12 +7,15 @@
  *
  * @covers ::maybe_hash_hex_color
  */
-class Tests_formating_maybeHashHexColor extends WP_UnitTestCase {
+class Tests_Formatting_maybeHashHexColor extends WP_UnitTestCase {
```

Comment:
```suggestion
class Tests_Formatting_MaybeHashHexColor extends WP_UnitTestCase {
```
Test classes should be in TitleCase. [ref](https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#test-classes).

🔢 Applies to other test class names in the PR, including those beginning with `wpSomething`, which per the standard should be `WpSomething`. There's inconsistency within the test suite on this, but the standard is clear, so I think it's better to go with the standard here.

Reply:
Related: [56846](https://core.trac.wordpress.org/ticket/56846)